doc/rados/operations/health-checks.rst

```diff
@@ -946,14 +953,14 @@ specific OSDs.
 
 To change this, run a command of the following form:
 
-.. prompt:: bash $
+.. prompt:: bash #
```

Comment:
Are you certain these commands require root privileges?

Reply:
I have not tested this myself but there are dozens of `ceph config` commands in the docs and almost all of them list the privileged prompt. For example: https://docs.ceph.com/en/latest/releases/reef/#upgrading-non-cephadm-clusters

I would be very surprised if user with no privileges could command the cluster with the `ceph` command, but I do not have anything better than circumstantial evidence about this.

Reply:
Your call, but:
I opened https://docs.ceph.com/en/tentacle/rados/operations/health-checks/
Starting from its first page: 'root' commands have an explicit 'sudo' mentioned. All the rest of the commands there have '$' as the prompt.